app/assets/stylesheets/flash.scss

```diff
@@ -0,0 +1,27 @@
+.flash {
+  padding: 8px 35px 8px 14px;
+  margin-bottom: 18px;
+  color: #c09853;
+  text-shadow: 0 1px 0 rgba(255, 255, 255, 0.5);
+  background-color: #fcf8e3;
+  border: 1px solid #fbeed5;
+  border-radius: 4px;
+
+  &--notice {
+    color: #468847;
+    background-color: #dff0d8;
+    border-color: #d6e9c6;
+  }
+
+  &--alert {
+    color: #b94a48;
+    background-color: #f2dede;
+    border-color: #eed3d7;
+  }
+
+  &--remind {
+    color: #3a87ad;
+    background-color: #d9edf7;
```

Comment:
Color literals like `#d9edf7` should only be used in variable declarations; they should be referred to via variable everywhere else.